src/generators/bendable-amogus/BendableAmogus.res

```diff
@@ -0,0 +1,239 @@
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+if black {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 1, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if rose {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 2, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if blue {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 3, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if cyan {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 4, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if yellow {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 5, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if pink {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 6, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if purple {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 7, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if orange {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 8, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if banana {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 9, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if coral {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 10, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if lime {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 11, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if green {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 12, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if gray {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 13, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if maroon {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 14, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if brown {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 15, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if tancolour {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 0, y: 1, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+  if white {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 1, y: 1, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
+
+
+
+
+
+Generator.drawTextureLegacy(
+    "Skin",
+    {x: 8, y: 8, w: 8, h: 8},
+    {x: 47, y: 229, w: 87-47, h: 253-229},
+    ~rotateLegacy=0.0,
+    (),
+  ) 
+  Generator.drawTextureLegacy(
+    "Skin",
+    {x: 8+32, y: 8, w: 8, h: 8},
+    {x: 47, y: 229, w: 87-47, h: 253-229},
+    ~rotateLegacy=0.0,
+    (),
+  ) 
+ Generator.drawImage("Folds", (0, 0))
+
+
+  
+}
+
+let generator: Generator.generatorDef = {
+  id: id,
+  name: name,
+  thumbnail: None,
```

Comment:
Please add a thumbnail - it should be a 256 x 256 JPG image.

Reply:
Done